app/javascript/helpers/forms/SigninFormHelpers.jsx

```diff
@@ -7,6 +7,8 @@ const validationSchema = yup.object({
 });
 
 export const signinFormConfig = {
+  mode: 'onChange',
```

Comment:
Not a fan of onChange trigger for the Sign In form. I find it annoying.

Reply:
I personally love to feel the liveness and responsiveness of SPA but I don't mind the other modes either.
We should make the forms consistent in all ways giving the expected UX including validations.
There still be another option to trigger validations though we should test them all and discuss what to pick on the next meeting Inshallah.

Reply:
Can you put up a quick video of the difference?

Reply:
The validation triggers instantly instead of strictly when you press the submit button.
https://user-images.githubusercontent.com/43917914/163856994-0ab77fd3-56e2-4861-bf2b-c6e70d2336cf.mov

Reply:
Yeah I'm not sure I like this on the sign in form - I don't think I've seen it on any other app

Reply:
Alright, I'll change it.